src/rastervision/task/task.py

```diff
@@ -171,7 +169,6 @@ def predict_batch(predict_chips, predict_windows):
             new_labels = self.backend.predict(
                 np.array(predict_chips), predict_windows, tmp_dir)
             labels += new_labels
-            print('.' * len(predict_chips), end='', flush=True)
```

Comment:
We shouldn't get rid of  this, it's useful to have this output. I think it's fine to leave this as a print if we can't accommodate the `end=''` in the logger.